docs/_docs/tools/control-script.adoc

```diff
@@ -521,6 +521,50 @@ validate_indexes has finished with errors (listed above).
 Like `idle_verify`, index validation tool works correctly only if updates are stopped. Otherwise, there may be a race between the checker thread and the thread that updates the entry/index, which can result in a false positive error report.
 ====
 
+=== Check SQL Index Inline Size
+
+During the operation of the Ignite cluster it could have a different SQL index inline size on cluster nodes.
+For example, due to the `IGNITE_MAX_INDEX_PAYLOAD_SIZE` property value is different on cluster nodes. The difference
+of index inline size may lead to performance drop.
+
+The `check_index_inline_sizes` command validates the indexes inline size of given caches on all cluster nodes. The inline
+size of secondary indexes is always checked on a node join and the WARN message is printed to the log if differ.
+
+[tabs]
+--
+tab:Shell[]
+[source,shell]
+----
+# Checks that secondary indexes inline sizes are the same on all cluster nodes.
+control.sh|bat --cache check_index_inline_sizes
+----
+--
+
+The console output if the index inline sizes are different:
```

Comment:
```suggestion
If the index inline sizes are different, the console output is similar to the data below:
```

Reply:
A topic comes first, then comes a rheme (a new information about the topic).......